Dockerfile

```diff
@@ -21,6 +21,9 @@ LABEL maintainer="Hypothes.is Project and contributors"
 # Expose the default port.
 EXPOSE 5000
 
+# Start the web server by default
+CMD ["init-env", "supervisord", "-c" , "conf/supervisord.conf"]
```

Comment:
Similar to `EXPOSE`, this very rarely changes, if ever, so I would only suggest moving it if having it at the bottom of the file is somehow un-idiomatic.